.github/actions/dotcover-report/dist/index.js

```diff
@@ -0,0 +1,6300 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
not part of dotCover - the action code is in `dotcover-report/index.js` and is "packed" by Node into `dotcover-report/dist/index.js` because that's apparently how you write actions that are composed of a JS script. not exactly pretty indeed but that is apparently "the" way to write your own custom action...